### PR TITLE
Fix structure of event API messages

### PR DESCRIPTION
**Pull request by shal**

### CODE REVIEW

File: app/models/label.rb [hunk 1]
```diff
@@ -50,12 +50,19 @@ def update_level_if_label_defined
     send_document_review_notification if key == 'document'
   end
 
-  # TODO: fix it when EventAPi will be added
+  def as_json_for_event_api
```
Comment (ec) [inline app/models/label.rb:58]: { 
      id: id,
      user: user.as_json_for_event_api,
      key: key,
      value: value
}

File: docs/event_api.md [hunk 2]
```diff
@@ -72,94 +72,117 @@ The fields `foo`, `bar`, `baz`, `qux` (example) are fields which carry useful in
...
     }
-    name: "model.document.created"
   }
+  name: "model.document.created"
+}
 ```
 | Field      | Description                         |
 | ---------- | ----------------------------------- |
-| `record`   | Created document with user up-to-date attributes.  |
+| `user`     | The up-to-date user attributes.     |
```
Comment (ec) [inline docs/event_api.md]: so, there is some "record" here, not only `user`, so please add in desctiption

File: docs/event_api.md [hunk 3]
```diff
@@ -72,94 +72,117 @@ The fields `foo`, `bar`, `baz`, `qux` (example) are fields which carry useful in
...
-  user: {
-    uid: "ID739065AFD3",
-    email: "example@barong.io",
-    role: "member",
-    level: 0,
-    otp: false,
-    state: "pending",
-    created_at: "2019-01-28T09:03:50Z",
-    updated_at: "2019-01-28T09:03:50Z"
+  record:
```
Comment (ec) [inline docs/event_api.md]: missing `{`

File: docs/event_api.md [hunk 4]
```diff
@@ -170,85 +193,139 @@ event: {
...
+## Format of `system.document.rejected` event\
+
 ```ruby
-  event: {
-    uid: "IDC554ED1D0F",
-    email: "example@barong.io",
-    name: "system.document.rejected"
-  }
- ```
+fevent: {
```
Comment (ec) [inline docs/event_api.md]: `fevent` ?

File: docs/event_api.md [hunk 5]
```diff
@@ -170,85 +193,139 @@ event: {
...
+    },
+    id: 1,
+    key: "something",
+    value: "verified"
+  },
+  name: "system.document.verified"
+}
+```
+
+## Format of `system.document.rejected` event\
```
Comment (ec) [inline docs/event_api.md]: `\`

**Review by ec — CHANGES_REQUESTED** [review]

File: app/models/label.rb (hunk 1 above)
Comment (shal) [reply app/models/label.rb:58]: why?

**Review by ec — APPROVED** [review]

**Pull request merged**